breeze-for-qubes/breeze-for-qubes.spec

```diff
@@ -0,0 +1,309 @@
+%if 0%{?qubes_builder}
+%define _sourcedir %(pwd)/breeze-for-qubes
+%define _builddir %(pwd)/breeze-for-qubes
+%define breeze_epoch %(cat epoch_breeze)
+%define breeze_version %(cat version_breeze)
+%else
+%define breeze_epoch 0
+%define breeze_version 5.5.4
+%endif
+
+%global         base_name   breeze
+
+%global         build_kde4  1
+
+Name:    plasma-breeze
```

Comment:
I am taking the suggested approach. I added both of Conflicts and Provides for the main package and the sub-packages; I hope this (Conflicts + Provides) is fine.